Signal/src/ViewControllers/NotificationSoundsViewController.h

```diff
@@ -1,9 +0,0 @@
-//
-//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
-//
-
-#import "OWSTableViewController.h"
-
-@interface NotificationSoundsViewController : OWSTableViewController
```

Comment:
Renamed this and generalized it to handle notification and ringtone selection, globally and per-thread.